### DISCUSSION TITLE
Previewer integration with `render-markdown.nvim` | `markview.nvim`

### DISCUSSION THREAD

**Discussion by ibhagwan** [ibhagwan/fzf-lua#1546] [Show and tell]
Absolutely magical, didn't realize how useful `:FzfLua blines` will be with this integration serving as a side-by-side:

> `:FzfLua blines`
![image](https://github.com/user-attachments/assets/a2c22d39-b916-47b9-abed-d5c47fd89bf5)
> `:FzfLua blines winopts.treesitter=true` (**treesitter enabled on main window**)
![image](https://github.com/user-attachments/assets/ff8b1513-aed2-484f-846a-e7d528fa3a33)

Ty @MeanderingProgrammer for the help in making this happen!

**EDIT:** To those interested:
- [My neovim config](https://github.com/ibhagwan/nvim-lua)
- Colorscheme: [vim-nightfly-colors](https://github.com/bluz71/vim-nightfly-colors) by @bluz71
- My `render-markdown.nvim` setup:
> https://github.com/ibhagwan/nvim-lua/blob/a59090e161aa33b292ae6242b6661cab9b7097e2/lua/plugins/init.lua#L81-L99
```lua
      require("render-markdown").setup({
        file_types = { "markdown" },
        code = {
          sign = false,
          width = "block",
          right_pad = 4,
          position = "right",
        },
        heading = {
          sign = false,
          -- icons = {},
        },
      })
```
- My `render-markdown.nvim` highlights:
> https://github.com/ibhagwan/nvim-lua/blob/a59090e161aa33b292ae6242b6661cab9b7097e2/lua/autocmd.lua#L73-L79
```lua
        vim.api.nvim_set_hl(0, "RenderMarkdownCode", { default = false, link = "DiffChange" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH1Bg", { default = false, link = "Visual" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH2Bg", { default = false, link = "DiffText" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH3Bg", { default = false, link = "DiffAdd" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH4Bg", { default = false, link = "DiffAdd" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH5Bg", { default = false, link = "DiffAdd" })
        vim.api.nvim_set_hl(0, "RenderMarkdownH6Bg", { default = false, link = "DiffAdd" })
```

**Comment by MeanderingProgrammer** [comment]
Huh, that is a really interesting example, ends up functioning like a searchable split view. Happy to help and loving the results!

**Reply by ibhagwan** [reply]
> Huh, that is a really interesting example, ends up functioning like a searchable split view. Happy to help and loving the results!

I was surprised too, I didn’t imagine this utility until playing around with it, an unintended “bonus”.

**Comment by ibhagwan** [comment]
Post https://github.com/ibhagwan/fzf-lua/commit/a5698761ff2b12a0d04cd0be0b60ae6970b0c539, `markview.nvim` is also supported, ty @OXY2DEV!

> **NOTE:** Prior to markview v25, this requires using the `dev` branch

![image](https://github.com/user-attachments/assets/5ff4bb63-0b57-413b-a6c4-b218b865b1b7)
